connector-j-5/pom.xml

```diff
@@ -18,21 +18,19 @@
     </description>
 
     <dependencies>
+
```

Comment:
Do we still need these changes (additional blank lines) here? Up to you, of course.